backend/de.metas.serviceprovider/src/main/java/de/metas/serviceprovider/issue/importer/IssueImporterService.java

```diff
@@ -249,6 +253,10 @@ private IssueEntity mergeIssueInfoWithEntity(@NonNull final ImportIssueInfo impo
 				? importIssueInfo.getMilestone().getMilestoneId()
 				: null;
 
+		final Status status = INVOICED.equals(existingEffortIssue.getStatus())
```

Comment:
pls document why we keep INVOICED status, but in other cases, we are using the status from ImportIssueInfo